test/JDBC/input/views/sys-sp_databases-dep-vu-verify.sql

```diff
@@ -9,3 +9,6 @@ go
 
 select * from sys_sp_databases_dep_vu_prepare_v1
 go
+
+EXEC sp_databases_dep_vu_prepare_PROC1
```

Comment:
should we also drop this proc?

Reply:
It is being dropped when db is getting dropped in cleanup file.